services/api-server/src/simcore_service_api_server/api/dependencies/application.py

```diff
@@ -13,17 +10,6 @@ def get_settings(request: Request) -> ApplicationSettings:
     return settings
 
 
-async def get_product_name(app: Annotated[FastAPI, Depends(get_app)]) -> str:
-    if not hasattr(app.state, "default_product_name"):
-        # lazy evaluation
-        async with app.state.engine.acquire() as conn:
-            app.state.default_product_name = await get_default_product_name(conn)
```

Comment:
ok so it was only here that we had this. Great, then the change i think it is pretty effective!
thx